### ISSUE TITLE
Unable to run several themes

### ISSUE THREAD

**Issue by erotavlas** [gatsbyjs/gatsby#21129]
I don't think this is a conincidence, because I tried one theme and one starter and both contain error when building.

For example see this

After going through the instruction (starting with  the gatsby starter theme) I get  failed building development bundle error
https://github.com/scttcper/gatsby-theme-casper/issues/7#issue-558600922

Furthermore If you look through the output there is an error relating GraphQL.  Which I am also getting when trying to use another starter below

See
https://stackoverflow.com/questions/60019273/error-85910-graphql-multiple-root-queries-found-primarytag-and-primaryt

Can anyone shed any light into this?  I assume these worked at some point, but no longer.

**Comment by LekoArts** [comment]
Thank you for opening this!

We don't maintain the themes in question so please reach out to the authors as it's most likely a bug on their side, not Gatsby's. At least I can't reproduce such behavior with our starters/themes. Thanks!

We're marking this issue as answered and closing it for now but please feel free to comment here if you would like to continue this discussion. We also recommend heading over to our [communities](https://www.gatsbyjs.org/contributing/community/#where-to-get-support) if you have questions that are not bug reports or feature requests. We hope we managed to help and thank you for using Gatsby!

**Comment by erotavlas** [comment]
I found the issue was occurring when I run the `gatsby develop` command from a Git Bash in Windows.  When I tried again from a regular command prompt the them I was using works fine.  

Try it with this theme

`gatsby new gatsby-starter-nehalem https://github.com/nehalist/gatsby-starter-nehalem`

then

```
cd gatsby-starter-nehalem
gatsby develop
```

Once from Git bash, then again from windows command prompt and you will probably encounter the same issue.
